views.py

```diff
@@ -654,7 +657,9 @@ def edit_annotation_base(request, image_pid, anno_pid, redirect_url):
         context_data.update(get_bound_edit_forms(annotation, AnnotationForm, PersonFormSet, InscriptionFormSet))
 
     image_link = 'https://%s/viewers/image/zoom/%s' % (BDR_SERVER, image_pid)
-    context_data.update({'image_link': image_link})
+    person_helper = PersonFormHelper()
```

Comment:
It looks like you can call the helper from within the formset.form
{% crispy example_formset example_formset.form.helper %}